src/main/kotlin/no/nav/sifinnsynapi/pleiepenger/syktbarn/ArbeidsgiverMeldingPDFGenerator.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.sifinnsynapi.pleiepenger.syktbarn
+
+import no.nav.sifinnsynapi.pdf.PDFGenerator
+import no.nav.sifinnsynapi.soknad.PleiepengerArbeidsgiverMelding
+import org.springframework.stereotype.Service
+import java.time.ZoneOffset.UTC
+import java.time.ZonedDateTime
+
+@Service
+class ArbeidsgiverMeldingPDFGenerator : PDFGenerator<PleiepengerArbeidsgiverMelding>() {
+
+    override val templateNavn: String
+        get() = "informasjonsbrev-til-arbeidsgiver"
+
+
+    override fun PleiepengerArbeidsgiverMelding.tilMap(): Map<String, Any?> = mapOf(
+        "arbeidsgiver_navn" to arbeidsgivernavn?.capitalize(),
+        "arbeidstaker_navn" to arbeidstakernavn.capitalize(),
+        "periode" to mapOf(
+            "fom" to DATE_FORMATTER.format(søknadsperiode.fraOgMed),
+            "tom" to DATE_FORMATTER.format(søknadsperiode.tilOgMed)
+        ),
+        "tidspunkt" to DATE_TIME_FORMATTER.format(ZonedDateTime.now(UTC))
+    )
+
+    override val bilder: Map<String, String>
```

Comment:
Kunne kanskje disse vært lagt inn i PDFGenerator? Er vel sjeldent vi vil trenge et spesielt symbol for kun en søknad?

Reply:
Har gjort det slik for å unngå å laste inn unødvendig bilder inn til templaten dersom pdf-en ikke trenger det. På denne måten trenger ikke pdf x å importere inn bilder som pdf y trenger.  Dette gjør pdf-genereringen raskere.
Det finnes selvfølgelig ingen andre behov for andre pdf-er pr. nå, men kommer sannsynligvis på sikt.

Reply:
Aha skjønner, godt poeng. Brukes alle disse i denne pdf nå da?

Reply:
Nei, egentlig ikke. Skal fjerne de som ikke brukes 👍